### ISSUE TITLE
AttributeError: 'ExtensionBundle' object has no attribute 'resourcesFolder'

### ISSUE THREAD

**Issue by aaronbell** [typedev/KernTool4#11]
I am trying to run KernTool4 (v`4.3.5` installed via Mechanic 2) in Robofont version 4.4 (build 2310101500) and when attempting to run the tool, I get an error in the output window:
```
Traceback (most recent call last):
  File "KernTool4.py", line 69, in <module>
AttributeError: 'ExtensionBundle' object has no attribute 'resourcesFolder'
```

Looks like an atribute has not been set properly in the file for the production mode:
```
if DEVELOP:
	pathForBundle = os.path.dirname(__file__)
	RESOURCES_FOLDER = os.path.join(pathForBundle, "resources")
	print(DEVELOP, RESOURCES_FOLDER)
else:
	kernToolBundle = mojo.extensions.ExtensionBundle("KernTool4")
	RESOURCES_FOLDER = str(kernToolBundle.resourcesFolder).replace('resources', 'lib/resources')
```

Setting `DEVELOP` to `True` gets past this bug.

**Comment by typedev** [comment]
Thanks for the note, already fixed, new version 4.3.6 should be updated in Mechanic

**Comment by aaronbell** [comment]
Verified it is working as expected 😄
